ml-agents/mlagents/trainers/ppo/optimizer.py

```diff
@@ -295,6 +295,9 @@ def _create_ppo_optimizer_ops(self):
         self.grads = self.tf_optimizer.compute_gradients(self.loss)
         self.update_batch = self.tf_optimizer.minimize(self.loss)
 
+    def initialize(self):
```

Comment:
if this method is only called in the __init__ do we need to make it a method? Can we just call `self.policy.initialize()` in the `__init__()`?

Reply:
legacy code when fixing tests. removed.